crates/static/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "powersync_static"
+edition.workspace = true
+version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+authors.workspace = true
+keywords.workspace = true
+
+[lib]
+name = "powersync"
+crate-type = ["staticlib"]
```

Comment:
I've added this as a separate package because building `powersync_loadable` as a dynamic library is not supported for watchOS builds (and `cargo build -p powersync_loadable` will build both a static and a dynamic library, I couldn't find a CLI flag to disable the dynamic build).

So I've created this new crate for static builds that we're now using for WASM and watchOS.